### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by infernalx** [f4rnham/BnS_pictures#2]
I tried to use ur tool but it seems its sending me to blank page, can u look into it ? Thanks

**Comment by Nevnto** [comment]
Yeah same thing with me i tried to upload but it gives me a blank page
